apps/emqx_conf/src/emqx_conf_schema.erl

```diff
@@ -865,7 +865,7 @@ fields("log_file_handler") ->
                     desc => ?DESC("log_file_handler_max_size")
                 }
             )}
-    ] ++ log_handler_common_confs(true);
+    ] ++ log_handler_common_confs(false);
```

Comment:
Q: this will make `emqx foreground` (used by the systemd service) stop logging to file, right?  Shouldn't the systemd service log to a file by default?  :thinking: 

This is fixed by these lines, I assume?

https://github.com/zhongwencool/emqx/blob/8243c93a2e5c4a984a5d8579df515e2c46e3e410/bin/emqx#L1106-L1107

Reply:
For a systemd service it's actually better to log to stdout/stderr only, as this stream will be captured by journald and stored centrally. You wouldn't need to care about disk space, log rotation, etc., as this stuff is already handled by journald.

Reply:
I remember that at least some customers (which use the .deb package installation) get their EMQX logs from `/var/log/emqx`, which is managed by EMQX. 

This change would make it no longer log to that location, it could be a change of mindset when someone needs to check the logs during a debug session?

Reply:
I just change the default handler from console to file. 
If user wants to file_handler, update emqx.conf is ok.
But before this PR:
It's impossible to change file(default) to console in emqx.conf with `./bin/emqx foreground`